src/components/molecules/TableHeader/TableHeader.tsx

```diff
@@ -119,60 +139,52 @@ const TableHeader: React.FC<TableHeaderProps> = ({
   }, [leaveSeat]);
 
   return (
+    // @debt move all style into TableHeader.scss
     <div className="row no-margin at-table table-header">
       <div className="header">
         <div className="back-button-container">
           <button
             type="button"
             title={"Leave " + seatedAtTable}
-            className="btn btn-primary back-button"
+            className="back-button"
             id="leave-seat"
             onClick={leaveSeat}
           >
-            Back
+            <FontAwesomeIcon
+              className="back-button-icon"
+              icon={faChevronLeft}
+              size="xs"
+            />
+            Leave table
           </button>
         </div>
         <div className="table-title-container">
-          <div
-            className="private-table-title"
-            style={{ fontSize: "16px", fontWeight: 700 }}
-          >
-            {tableOfUser?.title ?? "abc" /*seatedAtTable*/}
-            {tableOfUser && tableOfUser.capacity && (
-              <>
-                {" "}
-                <span
-                  style={{
-                    fontSize: "16px",
-                    marginLeft: "20px",
-                    fontWeight: 400,
-                  }}
-                >
-                  {usersAtCurrentTable &&
-                    `${
-                      tableOfUser.capacity - usersAtCurrentTable.length >= 1
-                        ? tableOfUser.capacity - usersAtCurrentTable.length
-                        : 0
-                    } seats left`}
-                </span>
-              </>
+          <div className="private-table-title">
+            {tableTitle}
+
+            {tableCapacity && (
+              <span className="private-table-seats-left">
+                {`${numberOfSeatsLeft} seats left`}
+              </span>
```

Comment:
Can't this be written like this?

```suggestion
              <span className="private-table-seats-left">
                {numberOfSeatsLeft} seats left
              </span>
```